groth16/src/r1cs_to_qap.rs

```diff
@@ -8,6 +8,38 @@ use r1cs_core::{ConstraintSystem, Index, SynthesisError};
 #[cfg(feature = "parallel")]
 use rayon::prelude::*;
 
+#[inline]
```

Comment:
This function could be worth moving to `r1cs-core` or `r1cs-std`?

Reply:
Yeah I also have a similar function in the Marlin codebase. I intend on doing a refactor of the CS infrastructure starting within a couple of weeks, so I’ll do this as part of that!